ross-nichols-solutions.js

```diff
@@ -0,0 +1,154 @@
+//PART ONE
+//#1
+const calculateCube = (num) => {
+    return num * num * num
+  }
+  
+  calculateCube(5)
+  console.log(calculateCube(5))
+  
+  //#2
+  const isAVowel = (str) => {
+    if(str.includes('a')) {
+      return true
+    } else if (str.includes('e')){
+      return true
+    } else if (str.includes('i')) {
+      return true
+    } else if (str.includes('o')) {
+      return true
+    } else if (str.includes('u')) {
+      return true
+    } else if (str.includes('y')) {
+      return 'sometimes'
+    } else {
+      return false
+    }
+  }
+  
+  isAVowel('a')
+  console.log(isAVowel('a'))
+  
+  //#3
+  const getTwoLengths = (str1, str2) => {
+    const arr = []
+    length1 = str1.length
+    length2 = str2.length
+    arr.push(length1)
+    arr.push(length2)
+    return arr
+  }
+  
+  getTwoLengths('Hank', 'Hippopopalous')
+  console.log(getTwoLengths('Hank', 'Hippopopalous'))
+  
+  //#4
+  const getMultipleLengths = (arr) => {
+    let wordCount = []
+    for(i = 0; i < arr.length; i++) {
+      len = arr[i].length
+      //console.log(len)
+      wordCount.push(len)
+    }
+    console.log(wordCount)
+  }
+  
+  getMultipleLengths(['hello','what','is','up','dude'])
+  
+  
+  
+  //PART TWO
+  //#1
+  const maxOfThree = (num1, num2, num3) => {
+    return Math.max(num1, num2, num3)
+  }
+  
+  maxOfThree(6, 9, 1)
+  console.log(maxOfThree(6, 9, 1))
+  
+  //#2
+  const printLongestWord = (arr) => {
+    let longest = arr[0]
+    for (let i = 1; i < arr.length; i++) {
+      if (longest.length <= arr[i].length) {
+        longest = arr[i]
+      }
+      //console.log(longest)
+    }
+    return longest
+  }
```

Comment:
nice! glad we got this working. Remember this pattern -- assume the first element in an array is the one you want, use a loop and conditional to check that first element against the rest of the elements, and finally replace the desired value as needed.